Source/Plugins/Core/com.equella.core/js/tsrc/mainui/Template.tsx

```diff
@@ -549,25 +575,39 @@ export const Template = React.memo(function Template({
     );
   }
 
-  const layout = useFullscreen({ fullscreenMode, hideAppBar }) ? (
-    <main>{children}</main>
-  ) : (
-    <div className={classes.appFrame}>
-      <AppBar className={classes.appBar}>
-        <Toolbar disableGutters>
-          {hasMenu && (
-            <IconButton
-              className={classes.navIconHide}
-              onClick={(_) => setNavMenuOpen(!navMenuOpen)}
-            >
-              <MenuIcon />
-            </IconButton>
-          )}
-          {titleArea()}
-          {menuArea()}
-        </Toolbar>
-        {tabs}
-      </AppBar>
+  function renderError(error: ErrorResponse) {
+    return (
+      <MessageInfo
+        open={errorOpen}
+        onClose={() => setErrorOpen(false)}
+        variant="error"
+        title={error.error_description ? error.error_description : error.error}
```

Comment:
either
```suggestion
        title={error.error_description ?? error.error}
```

or

```suggestion
        title={error?.error_description ?? error.error}
```

may work

Reply:
I'll leave as per is (existing code) for now, as I'm unclear as to if `error_description` is sometimes `""` which with the current code would then show the `error`.